pkg/apiserver/rest/dualwriter_mode1.go

```diff
@@ -27,36 +33,191 @@ func NewDualWriterMode1(legacy LegacyStorage, storage Storage) *DualWriterMode1
 
 // Create overrides the behavior of the generic DualWriter and writes only to LegacyStorage.
 func (d *DualWriterMode1) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	ctx = klog.NewContext(ctx, d.Log)
-	return d.Legacy.Create(ctx, obj, createValidation, options)
+	log := d.Log.WithValues("kind", options.Kind)
+	ctx = klog.NewContext(ctx, log)
+	var method = "create"
+
+	startLegacy := time.Now().UTC()
```

Comment:
[non-blocking] suggestion: there is no need to call `UTC()` method here. The usage of `time.Time` here is limited to monotonic clock usage (i.e. then calling `time.Since(startLegacy)` to obtain a duration), so calling `UTC()` is a nop, and may be misleading.